qa/suites/caasp/distros/sle_15.1.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/all/sle_15.1.yaml
```

Comment:
is there a reason why 15.1 is used instead of 15.2 ?

Reply:
it was started a while ago.
should I update it ?

Reply:
test it first, and if it works, update it?